src/components/common/infocolourBar/InfoColourbar.tsx

```diff
@@ -7,17 +7,29 @@ interface InfoColourBarType {
     Icon
     iconClass?: string
     renderActionButton?: () => JSX.Element
+    linkText?: string
+    redirectToLink?:() => void;
 }
 
-function InfoColourBar({ message, classname, Icon, iconClass, renderActionButton }: InfoColourBarType) {
+function InfoColourBar({ message, classname, Icon, iconClass, renderActionButton, redirectToLink, linkText }: InfoColourBarType) {
     return (
         <div className="info-bar-container">
             <div className={`${classname} info_text flex content-space pt-10 pb-10 pl-16 pr-16 br-4 top fs-13 fw-4`}>
                 <span className="flex top">
                     <div className="icon-dim-20 mr-10">
                         <Icon className={`${iconClass} icon-dim-20 mr-8`} />
                     </div>
-                    {message}
+                    <span>
```

Comment:
Change this to div and use span for the message, remove br & wrap a tag in another div.

Reply:
Done